wolfssl/wolfcrypt/sp_int.h

```diff
@@ -113,29 +113,33 @@ extern "C" {
     #error "Size of unsigned long not detected"
 #endif
 
-#if ULLONG_MAX == 18446744073709551615ULL
-    #define SP_ULLONG_BITS    64
+#ifdef ULLONG_MAX
```

Comment:
This solution looks good, but only solves a missing ULLONG_MAX. 

I suspect we will need to also do something for the other types. I don't think all systems properly support limits.h. And we may need to use the existing SIZEOF_LONG SIZEOF_LONG_LONG logic.

```
wolfssl/wolfcrypt/sp_int.h:66: #error "Size of unsigned short not detected" 
wolfssl/wolfcrypt/sp_int.h:89: #error "Size of unsigned int not detected" 
wolfssl/wolfcrypt/sp_int.h:137: #error "Size of unsigned long long not detected"
```

Also note spelling error below: `Detemine` -> `Determine`

Reply:
The others are defined in C89.